substrate/frame/revive/rpc/src/receipt_provider/db.rs

```diff
@@ -192,7 +192,21 @@ impl ReceiptProvider for DBReceiptProvider {
 		let latest_block =
 			U256::from(self.block_provider.latest_block_number().await.unwrap_or_default());
 
-		match (filter.from_block, filter.to_block, filter.block_hash) {
+		let from_block = match filter.from_block {
+			// Only latest tag supported
+			Some(BlockNumberOrTag::BlockTag(_)) => Some(latest_block),
```

Comment:
Shouldn't we rather error out in case an unsupported tag is used?

Reply:
yeah right let's do that